Jar/src/velma/task/Task.java

```diff
@@ -0,0 +1,52 @@
+package velma.task;
+
+public class Task {
+    protected String description;
```

Comment:
Its nice to see that you have clearly defined the accessibility of the variables and used clear variable names.